contents.json

```diff
@@ -4984,14 +4984,14 @@
         "official"
       ],
       "tags": [
-        "objc"
+        "swift"
       ],
       "description": "Official WordPress app",
       "source": "https://github.com/wordpress-mobile/WordPress-iOS",
       "itunes": "https://apps.apple.com/app/wordpress/id335703880",
-      "stars": 2747,
+      "stars": 18800,
```

Comment:
Star count is 2758 right now .. could you correct it? https://github.com/wordpress-mobile/WordPress-iOS

Reply:
Oh, sorry. I thought it was the number of stars in the App Store. Fixed. :)